include/deal.II/base/vectorization.h

```diff
@@ -966,6 +968,565 @@ vectorized_transpose_and_store(const bool                            add_into,
 
 #ifndef DOXYGEN
 
+#  if DEAL_II_VECTORIZATION_WIDTH_IN_BITS >= 128 && defined(__ARM_NEON__)
```

Comment:
Whichever preprocessor flag we settle for regading the header should also be set here.